config/storage.yml

```
@@ -0,0 +1,14 @@
+local:
+  service: Disk
+  root: <%= Rails.root.join('storage') %>
+
+test:
+  service: Disk
+  root: <%= Rails.root.join('tmp/storage') %>
+
+amazon:
+  service: S3
+  access_key_id: <%= ENV['AWS_ACCESS_KEY_ID'] %>
+  secret_access_key: <%= ENV['AWS_SECRET_ACCESS_KEY'] %>
+  region: 'us-west-2'
+  bucket: 'cwc-website'
```

```suggestion
  bucket: <%= ENV.fetch('AWS_BUCKET') { 'cwc-website' } %>
```

Can you change this to read from an ENV with a fallback default?